src/app/client.cljs

```diff
@@ -128,7 +128,7 @@
                                (comp/transact! this [(add-new-item {:item/id (random-uuid)})]))}
                    (dom/i :.plus.icon)))))))
 
-(def ui-item-list (comp/factory ItemList {:keyfn :item-list/all-items}))
+(def ui-item-list (comp/factory ItemList {:keyfn (constantly ::ui-item-list)}))
```

Comment:
this should not be...why this change?  The arg is a function that returns a key...if anything none is needed.  This particular change is actually quite bad, as it would cause the react key to change constantly.